radar/dev/typescript.yaml

```diff
@@ -1,16 +1,26 @@
 name: TypeScript
 logo: https://raw.githubusercontent.com/remojansen/logo.ts/master/ts.png
 blip:
-  - date: 2019-03-29
-    ring: TRIAL
+  - date: 2012-04-14
```

Comment:
Oh and fix the year to be 2021 😄

Reply:
oh yeah... I guess i am living in a past 😂